Unable to acquire the Scheduler while subscribing

```
Exception in thread "HmIP Network Key Thread" java.lang.IllegalStateException: Unable to acquire the Scheduler
	at org.osgi.util.pushstream.PushStreamProvider.acquireScheduler(PushStreamProvider.java:72)
	at org.osgi.util.pushstream.PushStreamProvider.createSimplePushEventSource(PushStreamProvider.java:550)
	at org.osgi.util.pushstream.PushStreamProvider.createSimpleEventSource(PushStreamProvider.java:491)
	at in.bytehue.messaging.mqtt5.provider.MessageSubscriptionProvider.subscribe(MessageSubscriptionProvider.java:125)
	at in.bytehue.messaging.mqtt5.provider.MessageSubscriptionProvider.replyToSubscribe(MessageSubscriptionProvider.java:114)
	at in.bytehue.messaging.mqtt5.provider.MessageReplyToPublisherProvider.publishWithReply(MessageReplyToPublisherProvider.java:129)
	at in.bytehue.messaging.mqtt5.provider.MessageReplyToPublisherProvider.publishWithReply(MessageReplyToPublisherProvider.java:119)
	at com.qivicon.runtime.grappa.wot.impl.MqttRequestDispatcherImpl.doRequest(MqttRequestDispatcherImpl.java:64)
	at com.qivicon.runtime.grappa.wot.impl.CloudConfigurationServiceImpl.putConfiguration(CloudConfigurationServiceImpl.java:109)
	at com.qivicon.security.homematicip.impl.NetworkKeyManager.storeInBackend(NetworkKeyManager.java:304)
	at com.qivicon.security.homematicip.impl.NetworkKeyManager.checkKeyInBackend(NetworkKeyManager.java:177)
	at com.qivicon.security.homematicip.HomematicIpSecurityComponent$NetworkKeyCreator.run(HomematicIpSecurityComponent.java:244)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.InterruptedException
	at java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireInterruptibly(AbstractQueuedSynchronizer.java:1220)
	at java.util.concurrent.locks.ReentrantLock.lockInterruptibly(ReentrantLock.java:335)
	at org.osgi.util.pushstream.PushStreamProvider.acquireScheduler(PushStreamProvider.java:59)
	... 12 more
```

--- a/in.bytehue.messaging.mqtt5.provider/src/main/java/in/bytehue/messaging/mqtt5/provider/MessageSubscriptionProvider.java
+++ b/in.bytehue.messaging.mqtt5.provider/src/main/java/in/bytehue/messaging/mqtt5/provider/MessageSubscriptionProvider.java
@@ -114,7 +114,7 @@ public final class MessageSubscriptionProvider implements MessageSubscription {
         return subscribe(null, subChannel, pubChannel, reference, true);
     }
 
-    private PushStream<Message> subscribe(
+    private synchronized PushStream<Message> subscribe(
 	            MessageContext context,
 	            final String subChannel,
 	            final String pubChannel,

